src/components/Header/Header.tsx

```diff
@@ -1,28 +1,74 @@
 import {Link, useLocation} from 'react-router-dom';
+import {useContext} from 'react';
+import cn from 'classnames';
 import {AppRoute} from '../../routing/AppRoute';
 import {Logo} from '../Logo/Logo';
 import {useAppDispatch, useAppSelector} from '../../hooks';
 import {getAuthStatus, getUser} from '../../store/auth/selectors';
 import {logOut} from '../../store/auth/auth';
+import {ThemeContext} from '../../services/ThemeProvider';
 import s from './Header.module.css';
 
 function Header() {
     const dispatch = useAppDispatch();
     const {pathname} = useLocation();
     const authStatus = useAppSelector(getAuthStatus);
     const userName = useAppSelector(getUser);
+    const {theme, toggleTheme} = useContext(ThemeContext);
 
     return (
-        <header className={`${s.header} bg-primary`}>
+        <header
+            className={cn(
+                s.header,
+                {'bg-primary': theme === 'dark'},
+                {'bg-info': theme === 'light'}
```

Comment:
```suggestion
                {'bg-primary': theme === 'dark'},
                {'bg-info': theme === 'light'}
```

За компонентой словарь тема: класс. Тут тогда словарь[theme].

Reply:
тут всё сделала по-другому, так что словарь не понадобился